tests/spec/fs.truncate.spec.js

```diff
@@ -192,3 +192,21 @@ describe('fs.truncate', function() {
     });
   });
 });
+
+describe('fsPromises.truncate', function () {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should error when length is negative', () => {
+
+    var fsPromises = util.fs().promises;
+    var contents = 'This is a file.';
+
+    return fsPromises.writeFile('/myfile', contents)
```

Comment:
Does this need to be a return @deepanjali19?

Reply:
It does, so that the containing test gets a Promise it can use to know when the test is complete.  With callback based tests you don't need to; but with Promise based ones you do.

Reply:
Thank you @humphd for explaining it better than I could have.